.gitignore

```diff
@@ -4,3 +4,4 @@
 /.tox
 /dist
 .vscode/
+.idea/
```

Comment:
don't touch gitignore in projects you don't own -- you may want to set up global gitignore in your personal dotfiles